flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1051,6 +1051,9 @@
     "com.vscodium.codium": {
         "finish-args-flatpak-spawn-access": "the app predates this linter rule"
     },
+    "com.vscodium.codium-insiders": {
+        "finish-args-flatpak-spawn-access": "the app predates this linter rule"
```

Comment:
The reason is not valid, this package did not predate the linter. Can you put a short line for the actual reason.

Reply:
thanks

Reply:
You are welcome!